StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheet+Link.swift

```diff
@@ -0,0 +1,163 @@
+//
+//  PaymentSheet+Link.swift
+//  StripePaymentSheet
+//
+
+import Foundation
+import UIKit
+@_spi(STP) import StripeCore
+
+// MARK: - Webview Link
+
+extension PaymentSheet: PayWithLinkWebControllerDelegate {
+
+    func payWithLinkWebControllerDidComplete(
+        _ PayWithLinkWebController: PayWithLinkWebController,
+        intent: Intent,
+        elementsSession: STPElementsSession,
+        with paymentOption: PaymentOption
+    ) {
+        let backgroundColor = self.configuration.appearance.colors.background.withAlphaComponent(0.85)
+        self.bottomSheetViewController.addBlurEffect(animated: false, backgroundColor: backgroundColor) {
```

Comment:
Seems like we should find paymentSheetViewController before presenting a blur effect, otherwise, users may end up in a bad UI state.

Reply:
Fixed!